common/src/main/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+
+
+swarm {
+    server.base.url = ${SERVER_URL}
+    server.base.url ="https://bumblebee.hive.swarm.space"
+}
+
+sendgrid.secret.key = ${SENDGRID_SERCRET_KEY}
+sendgrid.secret.key = "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"
```

Comment:
Is this just saying default value and otherwise read from environement varaible? I haven't used typesafe's ConfigFactory before so sorry for the silly question.

Reply:
No problem holden 😄 
Yes, it's a default value but it can be overridden while deploying services.
we will set the value for SENDGRID_SERCRET_KEY while deploying and sendgrid secret keys will be overridden.

We use this practice to keep develop and production environment variables separate.